packages/@romejs/compiler/lint/rules/react/jsxPropsNoSpreading.ts

```diff
@@ -0,0 +1,21 @@
+import {Path, TransformExitResult} from "@romejs/compiler";
+import {descriptions} from "@romejs/diagnostics";
+
+export default {
+	name: "reactJsxPropsNoSpreading",
+	enter(path: Path): TransformExitResult {
+		const {node} = path;
+
+		if (
+			node.type === "JSXElement" &&
+			node.attributes.some((attr) => attr.type === "JSXSpreadAttribute")
+		) {
```

Comment:
I think we'd want to point to the spread attribute instead of the element

```suggestion
		if (node.type === "JSXSpreadAttribute") {
```

Reply:
good point. that makes sense

Reply:
Oh I was thinking. The original reason for checking jsxelement first is so that it isn't just picking up other uses of spreading and only in jsx components